osgar/logger.py

```diff
@@ -38,13 +38,37 @@
 from ast import literal_eval
 import mmap
 
+g_logger = logging.getLogger(__name__)
 
 INFO_STREAM_ID = 0
 ENV_OSGAR_LOGS = 'OSGAR_LOGS'
 
 TIMESTAMP_OVERFLOW_STEP = (1 << 32)  # in microseconds resolution
 TIMESTAMP_MASK = TIMESTAMP_OVERFLOW_STEP - 1
 
+
+def format_header(start_time):
+    t = start_time
+    ret = []
+    ret.append(b'Pyr\x00')
```

Comment:
Why do you keep the header as multiple records?

Reply:
I wanted to keep the api same for both to allow stuff like this:
```python
        header = osgar.logger.format_header(datetime(2019, 1, 1, 1))
        packet = osgar.logger.format_packet(1, b"\x00"*10, timedelta())
        logdata = b"".join(header + packet)
```
Also the idea is that both return list of _atoms_ (for the lack of a better name). Those being the stuff you want to split into on parsing the log. That way it should be easier to write the test for parsing.